wide-github.css

```diff
@@ -35,13 +35,18 @@ header .container-lg {
   max-width: 100% !important;
 }
 .repository-content .inline-comments .comment-holder { /* Diff / code comments */
-  max-width: 960px !important;
+  max-width: inherit !important;
 }
 .repository-content .inline-comments .inline-comment-form-container {
-  max-width: 960px !important;
+  max-width: inherit !important;
 }
 .repository-content .inline-comments .inline-comment-form {
-  max-width: 960px !important;
+  max-width: inherit !important;
+}
+
+.repository-content #all_commit_comments .commit-comments-heading,
+.repository-content #all_commit_comments .comment-holder {
+  max-width: inherit !important;
```

Comment:
Which view is this for? I wasn't able to find anything out of place with just the inline comment changes.

Reply:
Click the View changes button in the PR and then go to the bottom GH editor. I'll attempt to back it out of my local copy and give a screenshot attached to this comment...

Reply:
Scratch that... it's at https://github.com/xthexder/wide-github/commit/46a4c85299099626ad16eca7d046173d1954a325 ... go to the bottom editor there.

Reply:
Ah I see it now. Changes all look good, I'll run the make.js and merge this when I'm home from work.